R/utils-lang.R

```diff
@@ -0,0 +1,148 @@
+
+# Return the ... arguments of the caller, as a list of quoted expressions.
+dot_exprs <- function() {
+  match.call(
+    definition  = sys.function(sys.parent()),
+    call        = sys.call(sys.parent()),
+    expand.dots = FALSE,
+    envir       = parent.frame(2L)
+  )$...
```

Comment:
This creeps me out. Changing to `[[`

```suggestion
  )[["..."]]
```